src/swing.ts

```diff
@@ -80,9 +80,10 @@ export interface Stack {
    * Creates an instance of Card and associates it with an element.
    *
    * @param {HTMLElement} element
+   * @param {boolean} prepend
    * @return {Card}
    */
-  createCard(elment: HTMLElement): void;
+  createCard(elment: HTMLElement, prepend: boolean): void;
```

Comment:
I believe it should be `createCard(elment: HTMLElement, prepend?: boolean): void;` so that prepend option is optional just like in the case of `swing` and it shouldnt break any existing app.